Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabDomUtils.cpp

```diff
@@ -428,14 +428,7 @@ namespace AzToolsFramework
                         {
                             patchesMetadata.m_shouldReloadContainerEntity = true;
                         }
-                        else
-                        {
-                            AZ_Warning(
-                                "Prefab", false,
-                                "A patch targeting '%.*s' is identified. Patches must be routed to Entities, Instances, or "
-                                "ContainerEntity.",
-                                AZ_STRING_ARG(patchPath));
-                        }
+                        // Skips other path prefix types (e.g. /LinkId/{someString}/).
```

Comment:
We still want this warning to show up if its targeting anything else other than linkId. So. add another condition here to check specifically for linkId and continue. If anything else is identified, we should still throw the warning.